### DISCUSSION TITLE
Hello everyone, I need your help: When I use apisix + opa plug-in for permission verification, the opa service cannot receive authentication requests from apisix

### DISCUSSION THREAD

**Discussion by Duanxinxin** [apache/apisix#8202] [Q&A]
The environment I use：
1. All services are deployed on k8s
2. apisix is deployed in the apisix namespace
3. opa services and business services are deployed in the dev namespace

When a request comes in, apisix uses the opa plug-in to verify the permission of the interface. After verification, apisix forwards the request to the corresponding business service。

But now my apisix cannot request to my opa service normally

The following is the configuration I made when I turned on the opa plug-in in apisix。

`"plugins": {
    "jwt-auth": {
      "disable": false
    },
    "opa": {
      "disable": false,
      "host": "http://opa.dev:8181",
      "policy": "ipim/dev/check_result",
      "ssl_verify": false
    },
    "proxy-rewrite": {
      "regex_uri": [
        "^/api/iam/(.*)",
        "/workspace/$1"
      ],
      "scheme": "http"
    }`

**Comment by tzssangglass** [comment]
> But now my apisix cannot request to my opa service normally
> 
> The following is the configuration I made when I turned on the opa plug-in in apisix。

Can you troubleshoot network problems yourself? Access the OPA service in the container running APISIX to see if the network is OK.
